services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -1036,6 +1040,17 @@ private OFAction actionSetOutputPort(final OFFactory ofFactory, final int output
         return actions.buildOutput().setMaxLen(0xFFFFFFFF).setPort(OFPort.of(outputPort)).build();
     }
 
+    /**
+     * Create an OFAction which sets the output port as in_port.
+     *
+     * @param ofFactory OF factory for the switch
+     * @return {@link OFAction}
+     */
+    private OFAction actionSetOutputInPort(final OFFactory ofFactory) {
```

Comment:
Too many duplication... I think it will be better to change type of second argument of `actionSetOutputPort` from `int` to `OFPort`. And use it for both cases.